index.html

```diff
@@ -45,6 +45,11 @@
       integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8"
       crossorigin="anonymous"
     ></script>
+    <!-- Open Graph Tags -->

```

Comment:
Can you put start it after the description of SEO is done? (line 28)

Reply:
done!